staging/src/k8s.io/apiserver/pkg/apis/audit/v1beta1/types.go

```diff
@@ -71,8 +71,8 @@ const (
 // Event captures all the information that can be included in an API audit log.
 type Event struct {
 	metav1.TypeMeta `json:",inline"`
-	// ObjectMeta is included for interoperability with API infrastructure.
-	// +optional
+
+	// Required. The name field under metadata is necesssary.
```

Comment:
Should we have it? It's unrelated to this change. Let's remove.

Reply:
Just to confirm, you mean to remove this comment for `..../v1beta1/types.go` and NOT for `...../v1alpha1/types.go`
(asking this because previously you mentioned to work on the v1beta1 thing )

> Also, please fix the v1beta1 api group.

Reply:
the audit event is completely unrelated to the API Event. Revert this file.

For v1beta1, I meant this one:
 staging/src/k8s.io/api/events/v1beta1/types.go

Reply:
oh alright.. got it